client/lib/products-values/translations.js

```diff
@@ -111,9 +111,15 @@ export const getJetpackProductsCallToAction = () => {
 			} ) }
 		</>
 	);
-	const search = translate( 'Get Search' );
-	const scan = translate( 'Get Scan' );
-	const antiSpam = <>{ translate( 'Get Anti-spam' ) }</>;
+	const search = isEnabled( 'plans/alternate-selector' )
+		? translate( 'Get Jetpack Search' )
+		: translate( 'Get Search' );
+	const scan = isEnabled( 'plans/alternate-selector' )
+		? translate( 'Get Jetpack Scan' )
+		: translate( 'Get Scan' );
+	const antiSpam = isEnabled( 'plans/alternate-selector' )
+		? translate( 'Get Jetpack Anti-spam' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Jetpack Anti-Spam' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/naming-consistency-across-cards)